lag/lag.yaml

```diff
@@ -4,28 +4,42 @@ components:
       x-include: 
       - '../common/common.yaml#/components/schemas/Named.Object'
       description: >-
-        The container for multiple LAG ports
+        The container for LAG (ports group) - aggregation of multiple LAG members (ports)
       type: object
       properties:
         ports:
           type: array
           items:
             $ref: '#/components/schemas/Lag.Port'
+          minItems: 1
+          maxItems: 32
+
+        protocol:
+          $ref: '#/components/schemas/Lag.Protocol'
+        min_links:
+          description: >-
+            Specifies the mininum number of member interfaces that must be active
+            for the aggregate interface to be available
+          type: integer
+          format: uint16
```

Comment:
`uint16` support is not there for generated sdks. Only int32&int64 are supported